docs/python/linting.md

```diff
@@ -8,278 +8,75 @@ DateApproved: 4/30/2022
-
-## Prospector
-
-### Command-line arguments and configuration files
-
-See [Prospector Command Line Usage](https://prospector.readthedocs.io/en/master/usage.html) for general options. For example, to set a strictness level of "very high," add the following line to your `settings.json` file:
-
-```json
-"python.linting.prospectorArgs": ["-s", "veryhigh"]
-```
-
-It's common with Prospector to use [profiles](https://prospector.readthedocs.io/en/master/profiles.html) to configure how Prospector runs. By default, Prospector loads the profile from a `.prospector.yaml` file in the current folder.
-
-Because Prospector calls other tools, such as Pylint, any configuration files for those tools override tool-specific settings in `.prospector.yaml`.  For example, suppose you specify the following, in `.prospector.yaml`:
-
-```yaml
-pylint:
-  disable:
-    - too-many-arguments
-```
-
-If you also have a `.pylintrc` file that enables the `too-many-arguments` warning, you continue to see the warning from Pylint within VS Code.
-
-### Message category mapping
-
-The Python extension maps all Prospector errors and warnings to the Error (E) category.
-
-## Flake8
-
-### Command-line arguments and configuration files
-
-See [Invoking Flake8](https://flake8.pycqa.org/en/latest/user/invocation.html) for general switches. For example, to ignore error E303 (too many blank lines), use the following setting:
-
-```json
-"python.linting.flake8Args": ["--ignore=E303"]
-```
-
-By default, Flake8 ignores E121, E123, E126, E226, E24, and E704.
-
-Flake8 user options are read from the `C:\Users\<username>\.flake8` (Windows) or `~/.config/flake8` (macOS/Linux) file.
-
-At the project level, options are read from the `[flake8]` section of a `tox.ini`, `setup.cfg`, or `.flake8` file.
-
-For details, see [Flake8 configuration](https://flake8.pycqa.org/en/latest/user/configuration.html).
-
-### Message category mapping
-
-The Python extension maps flake8 message categories to VS Code categories through the following settings. If desired, change the setting to change the mapping.
-
-| Flake8 category | Applicable setting<br/>(python.linting.) | VS Code category mapping |
-| --- | --- | --- |
-| F | flake8CategorySeverity.F | Error |
-| E | flake8CategorySeverity.E | Error |
-| W | flake8CategorySeverity.W | Warning |
-
-## mypy
-
-### Message category mapping
-
-The Python extension maps mypy message categories to VS Code categories through the following settings. If desired, change the setting to change the mapping.
-
-| mypy category | Applicable setting<br/>(python.linting.) | VS Code category mapping |
-| --- | --- | --- |
-| error | mypyCategorySeverity.error | Error |
-| note | mypyCategorySeverity.note | Information |
+Linters report issues with some predefined severity. This can be changed using `severity` setting for the linter. Refer to readme for the linter extension of your choice for more details on the supported values and severity levels.
```

Comment:
```suggestion
Linters report issues with some predefined severity. This can be changed using `severity` setting for the linter. Refer to each linter extension's README for more details on the supported values and severity levels.
```